src/Components/test/testassets/Components.TestServer/RazorComponents/Components/InteractiveStreamingRenderingComponent.razor

```diff
@@ -166,10 +181,13 @@ else
         return $"streaming-interactivity/update-state?state={stateJson}";
     }
 
-    private static string GetEnhancedNavigationUrl(ComponentState state, bool shouldStream)
+    private static string GetEnhancedNavigationUrl(ComponentState state, bool shouldStream, bool disableKeys)
     {
         var stateJson = GetEscapedStateJson(state);
-        return $"streaming-interactivity?{nameof(InitialState)}={stateJson}&{nameof(ShouldStream)}={shouldStream}";
+        return "streaming-interactivity" +
+            $"?{nameof(InitialState)}={stateJson}" +
+            $"&{nameof(ShouldStream)}={shouldStream}" +
+            $"&{nameof(DisableKeys)}={disableKeys}";
```

Comment:
I'm sure it works fine in these particular tests (since you'll have run them), but it's unusual to see this kind of URL construction without URL-encoding the values.

Reply:
`stateJson` comes from `GetEscapedStateJson()` which calls `Uri.EscapeDataString()`. The other parameters are just `bool`s. Is there something else I should be doing?

Reply:
Ah sorry, I didn't see that. Looks correct then.